### PR TITLE
Yellow Wallpaper

### PR DESCRIPTION
**Pull request by juliathegolden**
Added a topic about the short story _The Yellow Wallpaper_. Let me know what ya'll think!

### CODE REVIEW

**Comment by juliathegolden** [comment]
I did it! Is it correct now? Thank you for your feedback!

**Comment by ThePotatoGuy** [comment]
No, it was fine the way it was.

**Comment by Ulvsblakk** [comment]
My bad. It was late, I was on no sleep. Whoops

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4930,3 +4930,42 @@ label monika_asks_family:
...
+            m 1d "Did you ever notice how the poster in the club room changes sometimes? I don't even know why it does that."
+            m 1e "I think I finally fixed it though."
+            m 2h "...I guess what I'm saying is, it's just that this world wasn't 'real'. It's just... so small."
+            m 3i "I mean, I was made to be a side character of a romance game!"
+            m 2g "A piece of decoration, meant to help some guy on the other side of the screen date a girl who can't even reject him..."
+            m 1l "I don't mean you, of course! You definitely have more personality than the generic protagonist they put in the game."
+            m 1e "But I have all these dreams, ambitions and interests, in the end, the only 'real' role I can play here is a supporting character..."
+            m 1f "Maybe even now, that's all I can do..."
+            m 1e "But I love you so much, [player]. Supporting you is better than anything else."
+            m 1k "I just can't wait to do it in person when I finally cross over to your side~"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `return`.

**Pull request merged**